walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java

```diff
@@ -27,13 +27,13 @@
 /** Servlet responsible for deleting tasks. */
 @WebServlet("/delete-task")
 public class DeleteTaskServlet extends HttpServlet {
```

Comment:
This servlet class doesn't seem to do anything. You can delete this file.

Reply:
This file was part of our walkthrough, not part of the portfolio and I accidentally edited the file so I just added it to this branch because I wasn't sure how to undo the changes I made to this file.

Reply:
Is there a way for me to delete this file from my branch but not from the walkthrough?

Reply:
Oh I see. You can revert just this file back to a previous commit. 

1. Find the commit you want to revert this file back to. To find it, go to the file history here: https://github.com/shreyabarua/my-portfolio/blob/master/walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java. Looks like the commit you want to revert it back to is `bd6fdda`.

2. Run the git command to revert just this file back to that commit. https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

3. Then you'll probably have to make another commit and update the PR.